### DISCUSSION TITLE
💬 Chat 聊天

### DISCUSSION THREAD

**Discussion by lingbopro** [lingbopro/material-me#1] [Official Discussions]
# 💬 Chat 聊天

This discussion is for chat.

You can discuss anything here :)

For suggestions or feedback, go to [Suggestions](https://github.com/lingbopro/material-me/discussions/2) or [Feedback](https://github.com/lingbopro/material-me/discussions/3)

此讨论用于聊天。

你可以在这里讨论任何事情 :)

如需建议或反馈，请前往 [建议](https://github.com/lingbopro/material-me/discussions/3) 或 [反馈](https://github.com/lingbopro/material-me/discussions/2)

**Comment by Minemetero** [comment]
你这个库还在开发的时候就3star了？？？？？？？？？？？？？

**Reply by lingbopro** [reply]
emmm...好像是的

一个是我自己，一个是@zsr-lukezhang，还有一个和我互相follow的人

我让我bot也star一下就成4个了（bushi

**Reply by lingbopro** [reply]
今日目标已达成：清空工作树（）

**Comment by Minemetero** [comment]
干活aaaaaa干活啊

**Reply by lingbopro** [reply]
干着呢

今天已经推送了10个提交了

**Comment by Minemetero** [comment]
![Screenshot 2025-01-14 140735](https://github.com/user-attachments/assets/7f2e3ca1-4ab5-4a1a-b3af-01b71ab642d0)
![Screenshot 2025-01-14 140743](https://github.com/user-attachments/assets/718cd258-965e-40c4-8ce0-95891f044432)
hun?

**Reply by Minemetero** [reply]
pnpm 是坏了吗？

**Reply by Minemetero** [reply]
pnp

**Reply by lingbopro** [reply]
```shell
npm install -g pnpm
```

你用winget下的这个应该是独立版

另外得重开shell

**Reply by Minemetero** [reply]
![image_2025-01-15_102601757](https://github.com/user-attachments/assets/8480c16f-f63c-4552-aec9-1919184ce47c)
❓

**Reply by lingbopro** [reply]
`demo/` 这里面的东西得运行 `dev` 或者 `serve`，开 dev server 看

**Reply by lingbopro** [reply]
oh 这好像是我的问题

我在 fa64a0591e1c698eb4a84168e44b793969321460 里修了

更新一下分支试试？

**Reply by Minemetero** [reply]
:O

**Comment by Minemetero** [comment]
nani

**Comment by Minemetero** [comment]
![image_2025-01-30_110713161](https://github.com/user-attachments/assets/6a5aae88-3a95-459e-b9f7-3423d9add44c)
现在写commit都那么专业了？

**Reply by lingbopro** [reply]
AI写的，我改了改

**Reply by Minemetero** [reply]
oh~~~

**Comment by Minemetero** [comment]
你这变成烂尾工程了？

**Reply by lingbopro** [reply]
不是啊

**Reply by lingbopro** [reply]
最近在造这个项目要用到的轮子

**Reply by lingbopro** [reply]
新做的Tab组件我还得修修

**Reply by lingbopro** [reply]
> 最近在造这个项目要用到的轮子

轮子造完了，我可以回来继续搞了（

**Reply by Minemetero** [reply]
6 什么轮子 给我看看

**Reply by Minemetero** [reply]
什么轮子你造了一个月

**Reply by lingbopro** [reply]
> 什么轮子你造了一个月

其实轮子并不复杂，但是最近我一直在玩一个MC服务器  
现在那个服务器快倒闭了，我就可以专注于写代码了（（（

**Reply by Minemetero** [reply]
> > 什么轮子你造了一个月
> 
> 其实轮子并不复杂，但是最近我一直在玩一个MC服务器 现在那个服务器快倒闭了，我就可以专注于写代码了（（（

6

**Reply by Minemetero** [reply]
你这个项目是彻底烂尾了？:/ 你可以搞个sober fork
由社区维护的sober
heheheh

**Reply by lingbopro** [reply]
> 你这个项目是彻底烂尾了？:/ 你可以搞个sober fork 由社区维护的sober heheheh

真没烂尾，最近不是在搞Table-Tennis-Counter嘛

**Reply by Minemetero** [reply]
oh!

**Comment by Minemetero** [comment]
烂尾了，烂尾了，烂尾了

**Reply by lingbopro** [reply]
真没啊，重构已经马上完成了，没推送罢了

正在最后一个组件ripple上和lit斗智斗勇ing...

**Reply by Minemetero** [reply]
刑

**Reply by lingbopro** [reply]
我已经差不多重构完了，估计过会就可以推送上来

**Reply by Minemetero** [reply]
> 我已经差不多重构完了，估计过会就可以推送上来

那可太彳亍了啊

**Reply by lingbopro** [reply]
好了

#7
